### ISSUE TITLE
[1.19.2] Painting name collision with  Builders Addition (enderman)

### ISSUE THREAD

**Issue by vgrynch** [Darkhax-Minecraft/Dark-Paintings#16]
Please, change the name of the painting for Enderman. When include this mod together with the Builders Addition mod, no paintings from Builders Additions are included, with the server-side crashes like following:

```
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.IllegalArgumentException: Multiple entries with same value: Reference{ResourceKey[minecraft:painting_variant / buildersaddition:enderman]=net.minecraft.world.entity.decoration.PaintingVariant@1bc9f1c3}=DARKPAINTINGS_ENDERMAN and Reference{ResourceKey[minecraft:painting_variant / darkpaintings:enderman]=net.minecraft.world.entity.decoration.PaintingVariant@50a89074}=DARKPAINTINGS_ENDERMAN [in thread "Server thread"]
at com.google.common.collect.ImmutableMap.conflictException(ImmutableMap.java:376) ~[guava-31.0.1-jre.jar%23105!/:?]
```

OR, is there a way to change it or remove it using the data pack? If yes, please, provide an example for the workaround.

(Submitting this to both Builders Addition and Dark Painting mods issues - https://github.com/mrh0/buildersaddition/issues/76)

**Comment by Darkhax** [comment]
Hello,

Minecraft uses namespaced IDs for paintings, so `buildersaddition:enderman` and `darkpaintings:enderman` should not conflict in any way. Please include the version of Minecraft you are using, the versions of both mods, and the full crash log file.

**Comment by vgrynch** [comment]
Minecraft version is 1.19.2. 

Looks like you are right. This may be related to the server software I'm using (ArcLight). I made a clean install of Forge and only these 2 mods and no crash appeared.

However, I still could not get any paintings from the Builders Additions mod displayed.
When I include only the Builder Additions mod without Dark Painting, it also does not display new paintings, so it is probably not related to the Dark Paintings mod. 

Closing.
